File: dto/dtos/Language.cs
using dto.Paging;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace dto.dtos
{
    public class LanguageRes
    {
        public int LanguageId { get; set; }
        public string? LanguageCode { get; set; }
        public string? Name { get; set; }
    }
    public class LanguageReqEdit
    {
        [Required, MinLength(2)]
        public string? LanguageCode { get; set; }
        [Required, MinLength(2)]
        public string? Name { get; set; }
    }
    public class LanguageReqSearch : PagedReq
    {
    }
}

File: admin/src/dtos/Language.ts
import Common from "../utility/Common";
import { PagedReq } from "./PagedReq";

export interface LanguageRes {
  languageId?: string;
  languageCode?: string;
  name?: string;
}

export class LanguageReqEdit {
  languageCode?: string = "";
  name?: string = "";
}

export class LanguageReqSearch extends PagedReq {
  constructor(
    {
      pageNumber = 1,
      pageSize = Common.DEFAULT_PAGE_SIZE,
      orderBy,
      searchText = "",
    }: PagedReq,
    {}
  ) {
    super({
      pageNumber: pageNumber,
      pageSize: pageSize,
      orderBy: orderBy,
      searchText: searchText,
    });
  }
}